src/components/GlobeOptimized.jsx

```diff
@@ -492,12 +578,16 @@ export default function GlobeOptimized(props) {
 
     // segment star payload
     (stars || []).forEach((star) => {
-      const { location, mag } = star;
-      const adjustedMag = parseFloat(mag) * 10;
+      const { location, mag, color, id, name } = star;
+      const adjustedMag = parseFloat(mag) * 100;
       starCollection.push({
         ...location,
         size: starSizeScale(adjustedMag),
+        name: name,
+        color: color,
+        id: id
       });
+      starCoordinates.push(location.coordinates);
```

Comment:
Why do we need a separate array for star coordinations? I think we should already have it under `starCollection` ?

Reply:
I'll update it